e2e_playwright/lazy_loaded_modules.py

```diff
@@ -41,4 +41,9 @@
 
 for module in lazy_loaded_modules:
     loaded = module in sys.modules
-    st.write(f"**{module}**:", ("loaded" if loaded else "not loaded"))
+    st.write(f"**{module}**:", ("imported" if loaded else "not loaded"))
+
+if st.button("Import lazy loaded modules"):
```

Comment:
This is some preparation for a later potential test to check whether these modules exist.